lms/static/sass/discussion/views/_search.scss

```diff
@@ -16,3 +17,38 @@
     margin-top: 0;
   }
 }
+
+.forum-search > label {
+  position: absolute;
+  display: block;
+  color: #495057;
+  transition: all .1s ease-in-out;
+  padding-left: 10px;
+  padding-top: 10px;
+
+}
+
+.forum-search input::-webkit-input-placeholder {
```

Comment:
All these `.forum-search` can be combined and then all children of it like `input::-webkit-input-placeholder` and `input:-ms-input-placeholder` etc can then be listed together to assign `color: transparent` as
```
.forum-search{
.input::-webkit-input-placeholder,
.input:-ms-input-placeholder,
....
.input::placeholder{
color: transparent
}
}
```

Reply:
Thanks @noraiz-anwar for the context here.